paper-ripple-behavior.html

```diff
@@ -66,10 +66,10 @@
     /**
      * Ensures this element contains a ripple effect. For startup efficiency
      * the ripple effect is dynamically on demand when needed.
-     * @param {!Event=} opt_triggeringEvent (optional) event that triggered the
+     * @param {!Event=} optTriggeringEvent (optional) event that triggered the
      * ripple.
      */
-    ensureRipple: function(opt_triggeringEvent) {
+    ensureRipple: function(optTriggeringEvent) {
```

Comment:
Prefixing with opt_ deviates from the de facto convention that has fallen into place across the Polymer elements. However, the style guide asks that optional parameters be prefixed this way. Thanks for pointing this out, @danbeam.

We already use annotations in docs to inform the compiler of optional parameters. And, the style guide uses language that suggests that annotations are sufficient to communicate the semantics.

> Optional and variable arguments can also be specified in @param annotations. Although either convention is acceptable to the compiler, using both together is preferred.

I would like to propose that we maintain consistency now and perform a batched transformation on all elements at a later time. WDYT?